### PR TITLE
[12.0][ADD] mail_template_operating_unit

### PR DESCRIPTION
**Pull request by patrickrwilson**
This module adds operating units to email templates.

### CODE REVIEW

**Comment by patrickrwilson** [comment]
ping @max3903 @osi-scampbell

**Comment by patrickrwilson** [comment]
ping @max3903 @osi-scampbell

File: mail_template_operating_unit/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2020 Pavlov Media
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```
Comment (nikul-serpentcs) [inline mail_template_operating_unit/__init__.py]: Change License AGPL -> LGPL, everywhere 
Because In README file Added LGPL

Comment (patrickrwilson) [reply mail_template_operating_unit/__init__.py]: Done, Thanks

**Review by max3903 — APPROVED** [review]

**Review by osi-scampbell — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**